src/content/docs/en/guides/migrate-to-astro/from-gitbook.mdx

```diff
@@ -0,0 +1,75 @@
+
+- GitBook sites use a specialised version of Markdoc (a superset of Markdown) for content. Astro also supports Markdoc via the optional [Markdoc integration](/en/guides/integrations-guide/markdoc/) to use `.mdx` files in addition to standard Markdown files.
+
+
+## Switch from GitBook to Astro
+
+To convert a GitBook documentation site to Astro, start with our official docs theme starter template, or explore more community docs themes in our [theme showcase](https://astro.build/themes?search=&categories%5B%5D=docs).
+
+You can pass a `--template` argument to the `create astro` command to start a new Astro project with one of our official starters. Or, you can [start a new project from any existing Astro repository on GitHub](/en/install/auto/#starter-templates).
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npm create astro@latest -- --template docs
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm create astro@latest --template docs
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template docs
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+Once you have a new Astro project, you can sync your existing GitBook content to your new Astro project. GitBook has a [gitsync feature](https://docs.gitbook.com/product-tour/git-sync) that will automatically sync your GitBook content to a GitHub/GitLab repository. 
+
+To sync directly to the docs template's content collection, specify `src/content/docs/en` or `src/content/docs` as the project directory.
+
+:::caution
+When enabling git sync be sure to specify "**GitBook to GitHub**" as the priority. This will ensure that your GitBook content is synced to your GitHub repository. Otherwise, you will overwrite your existing GitBook content.
+:::
+
+After syncing the content, you will now have a copy of your GitBook content in your Astro repository. Disable git sync to prevent future syncing with GitBook.
+Note that although you now have your content migrated to your Astro project, it will not be immediately usable. To use this content in your Astro site, you will need to spend some time manually changing Gitbook's Markdoc into a format compatible with Astro. In particular:
+
+- Astro's [Markdoc integration](/en/guides/integrations-guide/markdoc/) requires that the file extension be `.mdoc`. This is to avoid conflicts with other Markdown extensions like `.mdx` and `.md`.
+- Gitbook uses a special syntax that differs from Markdoc where the `/` prefix denoting a closing tag is replaced with `end` for Gitbook files. You will need to update this notation throughout your files.
+- Some features of Gitbook rely on custom Markdoc components. These components will not exist in Astro and must be deleted from your Markdoc code.
```

Comment:
Here we should clarify the user has the option of actually creating these components after changing the tag syntax.
```suggestion
- Some features of GitBook rely on custom components. These components will not exist in Astro and must be created and added to your project through [Markdoc's config `tags` attribute](/en/guides/integrations-guide/markdoc/#render-markdoc-tags-as-astro-components) or removed from your files.
```